packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -68,12 +69,18 @@ import { SnapshotCreationCard } from './snapshot_creation_card';
 type ViewThreadPageProps = {
   identifier: string;
 };
+function useParams() {
```

Comment:
1. we usually use const instead of function
2. all those helpers/utils/hooks functions should always go outside of the component to utils.ts or helpers.ts or named file 
3. in this case this function is not really necessary, just do this in the component
```
  const [searchParams] = useSearchParams();
  const isEdit = searchParams.get('isEdit') ?? undefined;
```